lua/lualine/component.lua

```diff
@@ -108,21 +108,32 @@ function M:apply_highlights(default_highlight)
   end
 end
 
----apply icon in front of component (prepemds component with icon)
+---apply icon to component (appends/prepemds component with icon)
 function M:apply_icon()
   local icon = self.options.icon
   if self.options.icons_enabled and icon then
     if type(icon) == 'table' then
+      self.options.icon.align = icon.align
```

Comment:
What's this line suppose to do ? It seems you've just written 
```lua
x = x
```

Reply:
Nothing clearly, guess I made a poor assumption instead of properly reading.